features/app/components/AbstractApp.js

```diff
@@ -1,5 +1,14 @@
 import React, { Component } from 'react';
+
 import { roomSet } from '../../base/conference';
+import {
+    disposeLib,
+    initLib
```

Comment:
I'm not sure but... Didn't you choose the opposite of init to be destroy before? Did you change your mind & we no longer have the init-destroy pair anywhere?